mmapi.pm

```diff
@@ -183,8 +183,9 @@ sub get_job_autoinst_url ($target_id) {
     my $workers = $tx->res->json('/workers') // [];
     for my $worker (@$workers) {
         if ($worker->{jobid} && $target_id == $worker->{jobid} && $worker->{host} && $worker->{instance} && $worker->{properties}{JOBTOKEN}) {
-            my $hostname = $worker->{host};
-            my $token = $worker->{properties}{JOBTOKEN};
+            my $properties = $worker->{properties};
+            my $hostname = $properties->{WORKER_HOSTNAME} // $worker->{host};
```

Comment:
Thinking out loud here. Wasn't there a reason to avoid this in the first place?

Not saying it's wrong but I remember long conversations on (not) using FQDN or not being able to expect that it is what we want

Reply:
I think this code simply predates the introduction of `WORKER_HOSTNAME`.